airbyte-integrations/connectors/destination-pgvector/destination_pgvector/common/sql/write_strategy.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+"""Write strategy enum for SQL destinations."""
+
+from __future__ import annotations
+
+import enum
+
+
+class WriteStrategy(enum.Enum):
+    """Write strategy for SQL destinations."""
+    AUTO = "auto"
+    APPEND = "append"
+    REPLACE = "replace"
+    MERGE = "merge"
```

Comment:
Can we refactor to remove WriteStrategy and instead use destination write modes as defined in the airbyte protocol? (You should have a model that defines it?)